service/storage/prefixes.go

```diff
@@ -15,15 +15,19 @@
 package storage
 
 const (
-	prefixFirst        = 1
-	prefixLast         = 2
-	prefixHeader       = 3
-	prefixCommit       = 4
-	prefixEvents       = 5
-	prefixPayload      = 6
-	prefixHeight       = 7
-	prefixTransaction  = 8
-	prefixTransactions = 9
-	prefixCollection   = 10
-	prefixCollections  = 11
+	prefixFirst = 1
+	prefixLast  = 2
+
+	prefixHeightForBlock = 7
+
+	prefixCommit  = 4
+	prefixHeader  = 3
+	prefixEvents  = 5
+	prefixPayload = 6
+
+	prefixTransactionsForHeight = 9
+	prefixCollectionsForHeight  = 11
+
+	prefixTransaction = 8
+	prefixCollection  = 10
```

Comment:
Shouldn't we also change the numbers of the indexes to match the new order? We're not really in prod yet so now is the time to make breaking changes IMO.

Reply:
I don't want to change them now, because it will break all of the old indexes. We will do that on the first true release (at which point we will have to reindex everything).